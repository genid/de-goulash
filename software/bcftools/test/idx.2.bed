1	1172777	1172804
1	1172806	1172808
