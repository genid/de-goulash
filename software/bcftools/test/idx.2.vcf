##fileformat=VCFv4.1
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
1	1172783	.	CCTCTGTCCTGGTCGCGTTGCCCAGCG	C	.	.	.
1	1172784	.	C	T	.	.	.
