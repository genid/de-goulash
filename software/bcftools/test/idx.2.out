1	1172783	.	CCTCTGTCCTGGTCGCGTTGCCCAGCG	C	.	.	.
1	1172784	.	C	T	.	.	.
